Userland/Applications/HexEditor/HexEditor.cpp

```diff
@@ -421,6 +421,7 @@ void HexEditor::keydown_event(GUI::KeyEvent& event)
                 swap(m_selection_start, m_selection_end);
         } else
             m_selection_start = m_selection_end = m_position = new_position;
+
```

Comment:
Please avoid doing a formatting change if you're not modifying something close.
As this is improperly formatted regarding our rules, you can put that in its own commit if you want. (The rule says that if one branch require brackets, all branches should have some).